libs/backend/src/cast_vote_records/export.ts

```diff
@@ -569,8 +570,31 @@ async function randomlyUpdateCreationTimestamps(
   }
 }
 
+function getCastVoteRecordExportInProgressMarkerFilePath(
+  usbMountPoint: string
+): string {
+  return path.join(usbMountPoint, '.export-in-progress');
```

Comment:
nit: since we are writing to the filesystem and can't assume what other files are on the usb drive (or can we?), should we prefix this like `.vx-export-in-progress`?

Reply:
We can and can't haha - VxAdmin does provide a USB formatting tool that we encourage election officials to use, but you never know what might end up on a USB. A `vx-` prefix makes sense to me: https://github.com/votingworks/vxsuite/pull/4009/commits/5d77a902e2e83ded4ede18c1f73f295b0297196d